pkg/commands/app.go

```diff
@@ -517,6 +518,24 @@ func NewRootfsCommand() *cli.Command {
 	}
 }
 
+// NewSbomCommand is the factory method to add sbom command
+func NewSbomCommand() *cli.Command {
+	return &cli.Command{
+		Name:      "sbom",
+		ArgsUsage: "artifact",
+		Usage:     "generate sbom for an artifact",
+		Action:    artifact.SbomRun,
```

Comment:
In addition to usage, we may want to add a description since this subcommand is a bit complex.

```suggestion
		ArgsUsage: "ARTIFACT",
		Usage:     "generate sbom for an artifact",
		Description: fmt.Sprintf(`"ARTIFACT" uses a "transport":"details" format.

Supported transports:
  %s
`, artifact.ArtifactTypes.String()),
```